aspnet/mobile/native-mobile-backend.rst

```diff
@@ -1,11 +1,177 @@
+
+.. literalinclude:: native-mobile-backend/sample/ToDoApi/src/ToDoApi/Controllers/TodoItemsController.cs
+  :language: c#
+  :lines: 25-46
+  :dedent: 8
+
+Test adding new items using Postman by choosing the POST verb providing the new object in JSON format in the Body of the request. You should also add a request header specifying a ``Content-Type`` of ``application/json``.
+ 
+.. image:: native-mobile-backend/_static/postman-post.png
+
+The method returns the newly created item in the response.
+
+Updating Items
+^^^^^^^^^^^^^^
+
+Modifying records is done using HTTP PUT requests. Other than this change, the ``Edit`` method is almost identical to ``Create``. Note that if the record isn't found, the ``Edit`` action will return a ``NotFound`` (404) response.
+
+.. literalinclude:: native-mobile-backend/sample/ToDoApi/src/ToDoApi/Controllers/TodoItemsController.cs
+  :language: c#
+  :lines: 48-69
+  :dedent: 8
+
+To test with Poastman, change the verb to PUT and add the ID of the record being updated to the URL. Specify the updated object data in the Body of the request.
+
+.. image:: native-mobile-backend/_static/postman-put.png
+
+This method returns a ``NoContent`` (204) response when successful, for consistency with the pre-existing API.
+
+Deleting Items
+^^^^^^^^^^^^^^
+
+Deleting records is accomplished by making DELETE requests to the service, and passing the ID of the item to be deleted. As with updates, requests for items that don't exist will receive ``NotFound`` responses. Otherwise, a successful request will get a ``NoContent`` (204) response.
+
+.. literalinclude:: native-mobile-backend/sample/ToDoApi/src/ToDoApi/Controllers/TodoItemsController.cs
+  :language: c#
+  :lines: 71-88
+  :dedent: 8
+
+Note that when testing the delete functionality, nothing is required in the Body of the request.
+
+.. image:: native-mobile-backend/_static/postman-delete.png
+
+Common Web API Conventions
+--------------------------
+
+As you develop the backend services for your app, you will want to come up with a consistent set of conventions or policies for handling cross-cutting concerns. For example, in the service shown above, requests for specific records that weren't found received a ``NotFound`` response, rather than a ``BadRequest`` response. Similarly, commands made to this service that passed in model bound types always checked ``ModelState.IsValid`` and returned a ``BadRequest`` for invalid model types.
+
+Once you've identified a common policy for your APIs, you can usually encapsulate it in a :doc:`filter </mvc/controllers/filters>`. Learn more about `how to encapsulate common API policies in ASP.NET Core MVC applications <https://msdn.microsoft.com/en-us/magazine/mt767699.aspx>`_.
```

Comment:
I had included some links to additional resources that are planned, such as securing mobile APIs, performing offline sync, etc. However, since we don't have the content yet, @danroth27 didn't want dead links or pointers to non-existent content. I think we should plan to revisit this doc once we have some additional resources in place, and provide some additional references as you suggest.